src/app/shared/help-support/help-support.html

```diff
@@ -0,0 +1,25 @@
+<div class="docs-help-support">
+  <ul>
+    <li>
+      <a href="https://stackoverflow.com/questions/tagged/angular-material" target="_blank">
```

Comment:
```suggestion
      <a href="https://stackoverflow.com/questions/tagged/angular-material">
```
We don't use `target-"_blank"` anywhere on the site and using it without `rel="noopener"` causes Lighthouse Best Practice audit issues.